File: src/CrystalQuartz.Core/Domain/SchedulerData.cs
using CrystalQuartz.Core.Domain.Activities;

namespace CrystalQuartz.Core.Domain
{
    using System;
    using System.Collections.Generic;

    public class SchedulerData
    {
        public string Name { get; set; }

        public string InstanceId { get; set; }

        public IList<JobGroupData> JobGroups { get; set; }

        public SchedulerStatus Status { get; set; }

        public int JobsTotal { get; set; }

        public int JobsExecuted { get; set; }

        public IList<ExecutingJobInfo> InProgress { get; set; }

        public DateTime? RunningSince { get; set; }
    }
}

File: src/CrystalQuartz.Application.Client2/src/api/scheduler-data.ts
import { JobGroup } from './job-group';
import { RunningJob } from './running-job';
import { SchedulerEvent } from './scheduler-event';

export interface SchedulerData {
  Name: string;
  Status: string;
  InstanceId: string;
  RunningSince: number | null;
  JobsTotal: number;
  JobsExecuted: number;
  ServerInstanceMarker: number;
  JobGroups: JobGroup[];
  InProgress: RunningJob[];
  Events: SchedulerEvent[];
}
